setup.py

```diff
@@ -34,7 +34,7 @@
       license='BSD-2-Clause',
       keywords='cli vim vimwiki wiki',
       install_requires=[
-          'click~=7.1',
```

Comment:
Hi @Chickensoupwithrice thanks for the request! What would you think of changing this line to `click>=7.1`?

Reply:
yeah that works for me :) gimme a bit will update

Reply:
Done :3